consent/strategy_logout_test.go

```diff
@@ -164,7 +181,9 @@ func TestLogoutFlows(t *testing.T) {
 				require.NoError(t, err)
 				if sid != nil {
 					go func() {
-						sid <- *res.LoginSessionId
+						for i := 0; i < numSidConsumers; i++ {
```

Comment:
Not sure this is the best way to solve the issue of reading from `sid` channel 2 times in the headless logout tests, but this fixes the issue. What do you think @aeneasr?

Reply:
There probably is some go magic possible but I think this is fine :)